src/wp-admin/credits.php

```diff
@@ -12,15 +12,15 @@
 
 $title = __( 'Credits' );
 
-list( $display_version ) = explode( '-', get_bloginfo( 'version' ) );
+$display_version = get_bloginfo( 'classicpress_display_version' );
 
 include( ABSPATH . 'wp-admin/admin-header.php' );
 ?>
 <div class="wrap about-wrap full-width-layout">
 
 <h1><?php printf( __( 'Welcome to ClassicPress %s' ), $display_version ); ?></h1>
 
-<p class="about-text"><?php printf( __( 'Thank you for updating to the latest version! ClassicPress %s will smooth your design workflow and keep you safe from coding errors.' ), $display_version ); ?></p>
+<p class="about-text"><?php printf( __( 'Thank you for trying ClassicPress! We are under heavy development while we prepare for an initial release.' ), $display_version ); ?></p>
```

Comment:
And again for the change in `src/wp-admin/about.php`.

Reply:
can we close this?